match.qmd

```diff
@@ -2,25 +2,150 @@
 title: "Numerical Matching"
 ---
 
-## Do we need to match SAS numerically when using a different language?
+## What if open source languages and commercial software (such as SAS), do not match numerically with the analysis results they give?
 
--   What if we the same inputs yield similar, but numerically different results?
+-   What if the same inputs yield similar, but numerically different results?
 
--   What if we the same inputs yield drastically different results?
+-   What if the same inputs yield drastically different results?
 
--   What is the truth? Which is correct?
+-   What is the truth? Which is correct? Can any be considered 'correct'?
 
--   What if SAS and R are equivalent, but a third language yields numerical differences?
+## Historical Background
+
+The statistical analysis system (SAS), was developed by a collective of eight USA southern state universities in the late 1960's. The company SAS institute Inc. was founded in 1976, and their first product release of Base SAS, consisted of approximately 300,000 lines of code^1^. During the first 30 years of existence, SAS software became renowned in the highly regulated medical research industry, for its well documented methodology and its high quality, reproducible, robust and reliable analysis implementation. This made it the number one data analysis tool in the pharmaceutical industry and a gold standard for regulatory submissions around the world.
+
+R has available since 1993 with packages being added continuously, providing a continual development cycle and user-led package development^2^. This open-source language has a package repository called CRAN (Comprehensive R Archive Network) with over 21000 packages in it. The momentum and adoption of R in Pharma is growing, likely due to:
+
+1)  the wide range of additional functionality compared to SAS, which can lead to better efficiency,
```

Comment:
This is an opinion - are there any data to support? Do we want to position this as R vs SAS - or maybe, this is 'merits of SAS' and 'merits of R'

Reply:
Good point, definitely not wanting this as SAS vs R!  I'll reword